test/e2e/networking.go

```diff
@@ -171,7 +171,7 @@ var _ = Describe("Networking", func() {
 				DoRaw()
 		}
 
-		timeout := time.Now().Add(2 * time.Minute)
+		timeout := time.Now().Add(3 * time.Minute)
```

Comment:
The bump is to match the 2 minutes waitTimeout in the nettest container. This is to avoid the case where the container is still waiting for endpoint and the test already time out.